apertium/tagger.cc

```diff
@@ -0,0 +1,795 @@
+#include<apertium/tagger.h>
```

Comment:
Missing file header that was present in the old file. That could be why Git is getting confused.

Reply:
@Vaydheesh this still seems unresolved.

Reply:
can you tag the lines that needs to be added here

Reply:
Just the same copyright file header? There should also be a space after `#include`.